src/fingerprinting.js

```diff
@@ -25,6 +25,24 @@ function getFpPageScript() {
   // return a string
   return "(" + function (ERROR) {
 
+    // monkey patch captureStackTrace for 6to5 - this isn't a polyfill at all!
+    ERROR.captureStackTrace = Error.captureStackTrace || function (obj) {
```

Comment:
Same question for any other properties of `Error` below. Should probably use `ERROR` instead.